zk_toolbox/README.md

```diff
@@ -42,16 +42,18 @@ that connects all ZK chains, like the BridgeHub, the shared bridges, and state t
 To create a ZK Stack project, you must first create an ecosystem:
 
 ```bash
+# Skip this step if you are manually building from a local copy of the ZkSync repository (which is an existing ecosystem)
 zk_inception ecosystem create
 ```
 
-If you chose to not start database & L1 containers after creating the ecosystem, you can later run
-`zk_inception containers`
+If you chose to not start database & L1 containers after creating the ecosystem, or are manually building from a local
```

Comment:
I don't agree with this statement

Reply:
Whole statement or when manually building from local copy?

How would one run the database and containers if they don't create the ecosystem and opt in at that step?